packages/phone-number-privacy/signer/.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I left these out intentionally cause I suspect the signers won't be cloud functions (see https://github.com/celo-org/celo-monorepo/pull/3944). It'll depend on what we decide in today's infra meeting.

It's okay if you want to add them in for now, just an FYI

Reply:
Gotcha -- ya I had to add them to run it locally for testing